### PR TITLE
Fetch issues

### PR DESCRIPTION
**Pull request by 22phuber**

### CODE REVIEW

File: fixtures.sh [hunk 1]
```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
```
Comment (RicardoMonteiroSimoes) [inline fixtures.sh]: We shouldn't forget to remove this when the pushing and pulling of projects and issues is done

File: issue-tracker/src/components/main/main.component.jsx [hunk 2]
```diff
@@ -1,36 +1,71 @@
...
+
+    this.state = {
+      data: null
+    };
+  }
+
+  fetchProjects() {
+    if (
+      localStorage.getItem(client_uuid) &&
+      localStorage.getItem(client_uuid) !== ""
```
Comment (RicardoMonteiroSimoes) [inline issue-tracker/src/components/main/main.component.jsx:21]: If I remember correctly `if(localStorage.getItem(clien_uuid)` should be false if there is nothing there, much like when a parameter is null in normal JS

File: issue-tracker/src/components/newIssuePopup/newIssuePopup.component.jsx [hunk 3]
```diff
@@ -1,24 +1,28 @@
...
+import { faTimes } from "@fortawesome/free-solid-svg-icons";
 import './popup.styles.css';
 import Modal from "../modal/modal.component";
 import Form from "../form/form.component";
 import Input from "../input/input.component";
+import Button from "../button/button.component";
 
 
-const Popup = props => {
+const NewIssuePopup = props => {
```
Comment (RicardoMonteiroSimoes) [inline issue-tracker/src/components/newIssuePopup/newIssuePopup.component.jsx:11]: Will need to make something better out of this, maybe trough inheriting from a normal Popup? No idea... @ByRicardoSimoes

**Review by RicardoMonteiroSimoes — APPROVED** [review]
Good to merge, commented on things should be a future task for when the basics are running

File: issue-tracker/src/components/newIssuePopup/newIssuePopup.component.jsx (hunk 3 above)
Comment (22phuber) [reply issue-tracker/src/components/newIssuePopup/newIssuePopup.component.jsx:11]: Don't know if this is possible

**Pull request merged**